resources/js/Pages/Auth/Profile.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import {defineProps} from 'vue'
+import { router } from '@inertiajs/vue3'
+
+const props = defineProps<{
+  user: {
+    name: string
+    surname: string
+    email: string
+    school: {
+      name: string
+      }
+    }
+  errors: Record<string, string[]>
+  status?: string
+  }>()
+
+function submit() {
+  router.post('/profile/forgot-password')
+}
+</script>
+
+<template>
+  <div>
+    <p>Imię: {{ props.user.name }}</p>
+    <p>Nazwisko: {{ props.user.surname }}</p>
+    <p>Email: {{ props.user.email }}</p>
+    <p>Szkoła: {{ props.user.school.name }}</p>
+    <form @submit.prevent="submit">
+      <button type="submit">Zmiana hasła</button>
```

Comment:
When the user is logged into application, why he receives link to reset password? I believe that it should be a form, where he fills his old password and a new one and confirmation of the new one